client/question/services/FeedbackGeneratorService.js

```diff
@@ -24,18 +24,26 @@ tie.factory('FeedbackGeneratorService', [
   'RUNTIME_ERROR_FEEDBACK_MESSAGES', 'WRONG_LANGUAGE_ERRORS', 'LANGUAGE_PYTHON',
   'CLASS_NAME_AUXILIARY_CODE', 'CLASS_NAME_SYSTEM_CODE', 'PARAGRAPH_TYPE_TEXT',
   'PARAGRAPH_TYPE_CODE', 'PARAGRAPH_TYPE_SYNTAX_ERROR',
-  'PYTHON_PRIMER_BUTTON_NAME', 'ERROR_COUNTER_LANGUAGE_UNFAMILIARITY',
-  'ERROR_COUNTER_SAME_RUNTIME', 'UNFAMILIARITY_THRESHOLD',
-  'FEEDBACK_CATEGORIES',
+  'PARAGRAPH_TYPE_OUTPUT', 'PYTHON_PRIMER_BUTTON_NAME',
+  'ERROR_COUNTER_LANGUAGE_UNFAMILIARITY', 'ERROR_COUNTER_SAME_RUNTIME',
+  'UNFAMILIARITY_THRESHOLD', 'FEEDBACK_CATEGORIES',
   function(
     FeedbackObjectFactory, TranscriptService, ReinforcementGeneratorService,
     CODE_EXECUTION_TIMEOUT_SECONDS, SUPPORTED_PYTHON_LIBS,
     RUNTIME_ERROR_FEEDBACK_MESSAGES, WRONG_LANGUAGE_ERRORS, LANGUAGE_PYTHON,
     CLASS_NAME_AUXILIARY_CODE, CLASS_NAME_SYSTEM_CODE, PARAGRAPH_TYPE_TEXT,
     PARAGRAPH_TYPE_CODE, PARAGRAPH_TYPE_SYNTAX_ERROR,
-    PYTHON_PRIMER_BUTTON_NAME, ERROR_COUNTER_LANGUAGE_UNFAMILIARITY,
-    ERROR_COUNTER_SAME_RUNTIME, UNFAMILIARITY_THRESHOLD,
-    FEEDBACK_CATEGORIES) {
+    PARAGRAPH_TYPE_OUTPUT, PYTHON_PRIMER_BUTTON_NAME,
+    ERROR_COUNTER_LANGUAGE_UNFAMILIARITY, ERROR_COUNTER_SAME_RUNTIME,
+    UNFAMILIARITY_THRESHOLD, FEEDBACK_CATEGORIES) {
+
+    /**
+     * Object used to keep track of how many corrective feedback (values)
+     * were generated for a given test suite (keys).
```

Comment:
This is a bit vague -- "corrective feedback" could refer to buggy output feedback etc. We should pick a term for the type of feedback you're implementing and stick to it in all docstrings/varnames. (It's fine if the docstring ends up being a bit stilted, as long as it's unambiguous what it's referring to.)

Reply:
I meant "correctness" (typo). Is "correctness" OK? I don't have strong feelings about the name.